src/Voyager.php

```diff
@@ -68,12 +68,24 @@ public static function can($permission)
 
         if ($exist) {
             $user = User::find(Auth::id());
-            if ($user == null) {
+            if ($user == null || !$user->hasPermission($permission)) {
                 throw new UnauthorizedHttpException(null);
             }
-            if (!$user->hasPermission($permission)) {
-                throw new UnauthorizedHttpException(null);
+        }
+    }
+
+    public static function have($permission)
+    {
+        // Check if permission exist
+        $exist = Permission::where('key', $permission)->first();
+
+        if ($exist) {
+            $user = User::find(Auth::id());
```

Comment:
i think we wont need a console cmnd for that, just listen for the user logout event and flush any session data u want, which what laravel does by default
```php
// Illuminate\Foundation\Auth\AuthenticatesUsers Trait
public function logout(Request $request)
    {
        $this->guard()->logout();

        $request->session()->flush();

        $request->session()->regenerate();

        return redirect('/');
    }
```